src/shared/components/paginationControls/PaginationControls.tsx

```diff
@@ -82,9 +132,25 @@ export class TablePaginationControls extends React.Component<ITablePaginationCon
                     <Button key="prevPageBtn" disabled={!!this.props.previousPageDisabled} onClick={this.props.onPreviousPageClick}>
                         {this.props.previousButtonContent}
                     </Button>
-                    <Button key="textBetweenButtons" className={styles["auto-cursor"]} disabled={true}>
-                        {this.props.textBetweenButtons}
-                    </Button>
+                    <If condition={!this.textBoxActive}>
+                        <Then>
+                            <Button key="textBetweenButtons" className={styles["auto-cursor"]} disabled={!this.props.pageNumberEditable}
```

Comment:
hard to say without using it but the text box could probably just always be active.  styling would be easier that way.  in other words, no activation button